stroeer/coremedia/v1/curation-service.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package stroeer.coremedia.v1;
+
+import "stroeer/coremedia/v1/curated-item.proto";
+
+option java_multiple_files = true;
+option java_package = "de.stroeer.coremedia.v1";
+option go_package = "github.com/stroeer/tapir/go/stroeer/coremedia/v1;coremedia";
+
+service CurationService {
+  rpc GetCuratedList (GetCuratedListRequest) returns (stream stroeer.coremedia.v1.CuratedItem) {}
```

Comment:
notice the return type `stream` vs a Wrapper object.
this isn't actually a situation that `stream` is meant for, what do you say? wrapper or stream?